src/QueryGrammar.php

```diff
@@ -482,7 +482,7 @@ protected function compileWhereSearch(Builder $builder, array $where): array
     {
         $fields = '_all';
 
-        if (!empty($where['options']['fields'])) {
+        if (! empty($where['options']['fields'])) {
```

Comment:
Ooh! Is this a mistake, or are you adhering to PSR-2? :O Only because I actually quite like the space after a hashbang, but everyone else seems to think it looks revolting haha

Reply:
Yes! Love it! I want to bring it to the frontend